src/app/components/technical-record/technical-record.component.html

```diff
@@ -504,11 +504,11 @@ <h1 class="govuk-heading-xl title">Technical record</h1>
                 </tr>
                 <tr class="govuk-table__row" *ngFor="let axelNo of techRecordsJson.techRecord[0].axles">
                   <th class="govuk-table__cell govuk-!-font-weight-bold">Axle {{ isNullOrEmpty(axelNo.axleNumber) ? "-" : axelNo.axleNumber }}</th>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(axelNo.tyres.tyreCode) ? "-" : axelNo.tyres.tyreCode }}</td>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(axelNo.tyres.tyreSize) ? "-" : axelNo.tyres.tyreSize }}</td>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(axelNo.tyres.plyRating) ? "-" : axelNo.tyres.plyRating }}</td>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(axelNo.tyres.fitmentCode) ? "-" : axelNo.tyres.fitmentCode }}</td>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(axelNo.tyres.dataTrAxles) ? "-" : axelNo.tyres.dataTrAxles }}</td>
+                  <td [id]="'test-tyreCode-'+axelNo.axleNumber" class="govuk-table__cell">{{ isNullOrEmpty(axelNo.tyres.tyreCode) ? "-" : axelNo.tyres.tyreCode }}</td>
+                  <td [id]="'test-tyreSize-'+axelNo.axleNumber" class="govuk-table__cell">{{ isNullOrEmpty(axelNo.tyres.tyreSize) ? "-" : axelNo.tyres.tyreSize }}</td>
```

Comment:
typo replace [id] with id

Reply:
Here it is not a typo. Using [id] instead of id actually allows us to generate dynamic ids in an ngFor, because axelNo.axleNumber is an variable.